docs/vf-setup.md

```diff
@@ -0,0 +1,141 @@
+# Setting up Virtual Functions
+
+The SR-IOV Network Device Plugin requires SR-IOV virtual functions (VFs) to be created on the host ahead of startup. The device plugin does not manage the creation of virtual functions, and it does not automatically update when there is a change in the virtual functions. Each time the number of VFs or the driver used by the VF change the SR-IOV Network device plugin needs to be restarted. 
+This guide gives an overview for setting up SR-IOV virtual functions on a linux host for various NICs. 
+
+  * [Intel](#intel)
+  * [Mellanox](#mellanox)
+
+## Intel
+The below works on intel SR-IOV enabled cards including those from the Gigabit Ethernet, X710 and X722 families of Network Interface Cards. Note not all Gigabit Ethernet NICs are enabled with SR-IOV.
+
+Cards from the following families can be enabled through the sysfs endpoint as described below:
+* **Intel i40e Driver**
+	* Intel® Ethernet Controller X722 
+	* Intel® Ethernet Controller X710
+* **Intel ixgbe Driver** 
+	* Intel® Ethernet Controller 82599
+	* Intel® Ethernet Controller X520
+	* Intel® Ethernet Controller X540
+	* Intel® Ethernet Controller x550
+	* Intel® Ethernet Controller X552
+	* Intel® Ethernet Controller X553
+
+### Creating VFs with sysfs
+First select a compatible NIC on which to create VFs and record its name (shown as PF_NAME below). 
+
+To create 8 virtual functions run:
+```
+echo 8 >> /sys/class/net/${PF_NAME}/device/sriov_numvfs
+```
+To check that the VFs have been successfully created run:
+
+```
+lspci | grep "Virtual Function"
+``` 
+
+This method requires the creation of VFs each time the node resets. This can be handled automatically by placing the above command in a script that is run on startup such as `/etc/rc.local`.
+
+### Common issues
+Some cards using the `ixgbe` may require additional configuration. This is likely the case if the message `write error: Cannot allocate memory` is returned when creating VFs using the above method. This issue has been observed on the X552 NIC. 
+
+To resolve this issue, try setting the maximum allowed VFs in the driver config:
+
+``` 
+modprobe -r ixgbe; modprobe ixgbe max_vfs=3
+
+```
+Next set the VFs as above
+
+```
+echo 8 >> /sys/class/net/${PF_NAME}/device/sriov_numvfs
```

Comment:
I think this is a type here? This should be `>` instead of `>>`

Reply:
Thanks for catching that - fixed